Common subdirectories: frescobaldi-2.0.16.old/build and frescobaldi-2.0.16/build
Common subdirectories: frescobaldi-2.0.16.old/frescobaldi_app and frescobaldi-2.0.16/frescobaldi_app
Common subdirectories: frescobaldi-2.0.16.old/macosx and frescobaldi-2.0.16/macosx
diff -u frescobaldi-2.0.16.old/setup.cfg frescobaldi-2.0.16/setup.cfg
--- frescobaldi-2.0.16.old/setup.cfg	2012-02-05 07:08:24.000000000 +0100
+++ frescobaldi-2.0.16/setup.cfg	2014-10-24 15:08:48.141335620 +0200
@@ -1,6 +1,2 @@
-[bdist_wininst]
-bitmap=frescobaldi-wininst.bmp
-install-script=frescobaldi-wininst.py
-
 [sdist]
 force-manifest=1
